.gitignore

```diff
@@ -140,7 +140,11 @@ cython_debug/
 # asv
 Django/
 html/
-results/
+results/*
+!results/macos-12
+!results/ubuntu-22.04
```

Comment:
Are these the default system names we get when running in github actions?

Reply:
The system name changes between runs and the results for the same kind of machine will be added under different directories, so I set the name of the machines to their respective OS using 

```yaml
asv machine --machine ${{ matrix.os }} --yes
```

Reply:
Super explanation. Thank you.